make_docs.nu

```diff
@@ -254,7 +254,7 @@ $"($example.description)
         $'::: warning(char nl)Dataframe commands were not shipped in the official binaries by default, you have to build it with `--features=dataframe` flag(char nl):::(char nl)(char nl)'
     } else { '' }
 
-    let plugins = if $command.name in ['from ini', 'from ics', 'from eml', 'from vcf'] {
+    let plugins = if $command.name in ['from ini', 'from ics', 'from eml', 'from vcf', 'query', 'query xml', 'query json', 'query web'] {
```

Comment:
Thanks for fixing this!
The message below points to `nu_plugin_formats`. Could we make the message specific to `nu_plugin_query` as well?

Reply:
Sorry about that ! It should be fixed in the new commit. By the way, I could not generate the documentation using `make_docs.nu` directly so I'm not sure it compiles.